apps/web/src/routes/course/[courseName]/skill/[skillName]/introduction.svelte

```diff
@@ -68,6 +68,11 @@
         href={`course/${courseName}/skill/${practiceHref}`}
         >Practice {title}</Button
       >
+      <Button
+        style="primary"
+        href={`course/${courseName}`}
+        >Go back {title}</Button
```

Comment:
```suggestion
        >Go back to the course</Button
```

I would do it this way. And, I would also change the order of the buttons. What do you think? 

And the go back button should not be primary

Reply:
Agreed! I'm pushing these changes in a bit